### PR TITLE
Update Notes on iOS Simulator

### PR DESCRIPTION
**Pull request by steve-the-edwards**

### CODE REVIEW

File: workflow-core/README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 This module contains the core workflow APIs.
 
+## Kotlin Multiplatform
+
+This module is a Kotlin Multiplatform module. The targets currently included for build and test
+are `jvm`, `ios`, and `iosSimulatorSimulatorArm64`. If you are having issues with the tests,
+ensure you have the correct version of XCode installed and can launch a simulator as its specified
```
Comment (rjrjr) [inline workflow-core/README.md:9]:
```suggestion
ensure you have the correct version of XCode installed and can launch a simulator as it's specified
```

File: workflow-runtime/README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
 # Module workflow-runtime
 
 This module contains the core APIs and logic for running workflows.
+
+## Kotlin Multiplatform
+
+This module is a Kotlin Multiplatform module. The targets currently included for build and test
+are `jvm`, `ios`, and `iosSimulatorSimulatorArm64`. If you are having issues with the tests,
+ensure you have the correct version of XCode installed and can launch a simulator as its specified
```
Comment (rjrjr) [inline workflow-runtime/README.md:9]:
```suggestion
ensure you have the correct version of XCode installed and can launch a simulator as it's specified
```

**Review by rjrjr — APPROVED** [review]

File: RELEASING.md [hunk 3]
```diff
@@ -9,6 +9,10 @@
    (Note we exclude benchmarks, but you can check those too!)
    ```bash
    ./gradlew build && ./gradlew connectedCheck -x :benchmarks:dungeon-benchmark:connectedCheck -x :benchmarks:performance-poetry:complex-benchmark:connectedCheck -x  :benchmarks:performance-poetry:complex-poetry:connectedDebugAndroidTest -x :samples:todo-android:app:connectedDebugAndroidTest
+   ```
+   NOTE: If you have any unexpected errors in the build or tests and they are related to non-jvm
+   targets you may need to update your XCode or other iOS tools. See note in the workflow-core andq
```
Comment (steve-the-edwards) [inline RELEASING.md:14]:
```suggestion
   targets you may need to update your XCode or other iOS tools. See note in the workflow-core and
```

File: workflow-core/README.md [hunk 4]
```diff
@@ -2,6 +2,13 @@
 
 This module contains the core workflow APIs.
 
+## Kotlin Multiplatform
+
+This module is a Kotlin Multiplatform module. The targets currently included for build and test
+are `jvm`, `ios`, and `iosSimulatorSimulatorArm64`. If you are having issues with the tests,
```
Comment (bnvinay92) [inline workflow-core/README.md:8]: `iosSimulatorSimulatorArm64` is this a typo? Also, I thought we added js support too.

**Pull request merged**